nautobot/docs/additional-features/jobs.md

```diff
@@ -553,57 +553,113 @@ Provision of user input (`data` values) via the CLI is not supported at this tim
-
-        from django.conf import settings
+While individual methods within your Job can and should be tested in isolation, you'll likely also want to test the entire execution of the Job. Nautobot 1.3 introduced a couple of changes to the way Jobs are handled. As a consequence, the recommended job testing approach also differs between 1.2.x and 1.3.x and later.
+
+### 1.3.x and later
+
+The simplest way to test the entire execution of Jobs from 1.3 on is via calling the `nautobot.utilities.testing.run_job_for_testing()` method, which is a helper wrapper around the `run_job` function used to execute a Job via Nautobot's Celery worker process.
+
+Because of the way `run_job_for_testing` and more specifically `run_job()` works, which is somewhat complex behind the scenes, you need to inherit from `nautobot.utilities.testing.TransactionTestCase` instead of `django.test.TestCase` (Refer to the [Django documentation](https://docs.djangoproject.com/en/stable/topics/testing/tools/#provided-test-case-classes) if you're interested in the differences between these classes - `TransactionTestCase` from Nautobot is a small wrapper around Django's `TransactionTestCase`).
+
+### 1.2.x
+
+If your test case needs to be backwards-compatible with test execution against Nautobot 1.2.x, you need to handle a couple more things manually:
+
+- Set up the `"job_logs"` database correctly for testability
+  ```
+  from django.conf import settings
+
+  if "job_logs" in settings.DATABASES:
+      settings.DATABASES["job_logs"] = settings.DATABASES["job_logs"].copy()
+      settings.DATABASES["job_logs"]["TEST"] = {"MIRROR": "default"}
+  ```
+- Replicate the behavior of `run_job_for_testing` manually so that your test execution most closely resembles the way the celery worker would run the test and set up the `databases` field on the test class correctly
+  ```python
+  import uuid
+
+  from django.contrib.auth import get_user_model
+  from django.contrib.contenttypes.models import ContentType
+  
+  from nautobot.extras.context_managers import web_request_context
+  from nautobot.extras.jobs import run_job
+  from nautobot.extras.models import JobResult, Job
+
+  def run_job_for_testing(job, data=None, commit=True, username="test-user"):
+      if data is None:
+        data = {}
+      user_model = get_user_model()
+      user, _ = user_model.objects.get_or_create(username=username, is_superuser=True, password="password")
+      job_result = JobResult.objects.create(
+          name=job.class_path,
+          obj_type=ContentType.objects.get_for_model(Job),
+          user=user,
+          job_id=uuid.uuid4(),
+      )
+      with web_request_context(user=user) as request:
+          run_job(data=data, request=request, commit=commit, job_result_pk=job_result.pk)
+      return job_result
+  ```
+- Re-create the default Statuses on `setUp` in your test classes, because `django.test.TransactionTestCase` truncates them on every `tearDown`
```

Comment:
```suggestion
- Set up the `databases` field on the test class correctly, and re-create the default Statuses on `setUp` in your test classes, because `django.test.TransactionTestCase` truncates them on every `tearDown`:
```